docs/faq.rst

```diff
@@ -0,0 +1,126 @@
+        For example, in the US, the ZIP code where a person lives is well correlated with their
+        race.
+        Even if the model is not provided with race as a feature, the model will pick up on it
+        implicitly via the ZIP code (and other features).
+        Worse, without having the race available in the dataset, it is not possible to determine
+        whether any use of ZIP code was as a proxy for race, or for some other reason.
+
+    The model is unfair because the data are biased. Isn't it better to get better data?
+        Yes, but gathering unbiased data might not always be practical.
+        Fairlearn and similar packages offer techniques to redistribute harms between subgroups
+        in the data.
+
+    Won't making a model fairer reduce its accuracy?
+        Any technique which reduces disparity in some metric is most likely to impact other metrics.
+        However, we have found empirically that it is usually possible to obtain substantial reductions
+        in disparity (as measured by some suitable metric) at minimal cost in accuracy.
+        Indeed, for certain subgroups in the data, accuracy may be improved dramatically.
+
+.. topic:: Fairlearn Questions
+
+    Can Fairlearn make my model fair?
+       Naive use of Fairlearn is not going to result in fairer models.
+       Indeed, when used incorrectly, Fairlearn could produce model which is *less* fair.
+
+       What Fairlearn can do is generate a model which has lower disparity as measured by
+       a user-specified metric.
+       Whether this makes the model fairer is outside the remit of Fairlearn - or indeed of
+       machine learning in general.
+       To determine whether the new model is fairer, the whole system of which it is a part
+       needs to be considered in terms of the societal context in which it is used.
+
+    What sort of fairness related harms can Fairlearn address?
+        Fairlean is focussed on group fairness.
+        That is, minimising the difference in some metric evalulated on
+        specified subgroups in the data.
+        This is certainly not the only way of looking at fairness, though.
+
+    Can Fairlearn be used to detect bias in datasets?
+        Not yet....
+
+    Can Fairlearn recommend ways to make my model fairer?
+        No. Fairlearn is not magic.
+
+        Fairness is a social concept, and no technological solution can make
+        things fair automatically.
+        In order to use Fairlearn successfully, you first need to work out
+        what fairness means for your particular problem.
+        Once that is determined, you can then work out how to use the
+        various algorithms and fairness metrics which Fairlearn supports
+        to develop models which are *fairer by the standard you have chosen*.
+
+    What unfairness mitigation techniques does Fairlearn support?
+        ExpGrad, PostProcessing....
```

Comment:
This should probably link to the mitigation section where I put the table, otherwise we need to maintain multiple locations with all the techniques.

Reply:
That answer was not complete :-)